{"name": "test_incorrect[incorrect_data0]", "status": "passed", "attachments": [{"name": "log", "source": "e9f6f54e-5ecf-41af-ac66-91ec6eb27806-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '1303845892@qq.com', 'password': '', 'check': '密码不能为空'}"}], "start": 1594723045870, "stop": 1594723048223, "uuid": "db03eaef-1180-4dd8-b73b-bca811b156d2", "historyId": "e64c0af6fef26d85bcdee262d7a7479c", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "24464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}